deploy/env-development.json

```diff
@@ -117,6 +135,103 @@
                 "[resourceId('Microsoft.Network/virtualNetworks', 'dev-vnet')]"
             ]
         },
+        {
```

Comment:
do we want to put the vmss in a separate template?  If we don't it means we'll get CI VMSSes when we deploy to our dev subscription?

Reply:
What is the parameter `ReplicasCount` and keep the consistency of the code? Fewer deltas?